### PR TITLE
Feature/schedule-visualization

### PR DESCRIPTION
**Pull request by Ahmedhossamdev**
fixes: #123

### CODE REVIEW

File: src/components/oba/TripDetailsPane.svelte [hunk 1]
```diff
@@ -14,13 +15,6 @@
 	let interval;
 	let busPosition = 0;
 
-	function formatTime(seconds) {
```
Comment (aaronbrethorst) [inline src/components/oba/TripDetailsPane.svelte:17]: 👍

File: src/components/schedule-for-stop/scheduleAccordionItem.svelte [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<script>
```
Comment (aaronbrethorst) [inline src/components/schedule-for-stop/scheduleAccordionItem.svelte:1]: the title of this file should be `ScheduleAccordionItem.svelte`, right?

File: src/components/stops/StopPane.svelte [hunk 3]
```diff
@@ -106,12 +106,21 @@
...
 						{#if routeShortNames()}
 							<h2 class="h2 mb-0 text-white">{$t('routes')}: {routeShortNames().join(', ')}</h2>
 						{/if}
+						<div class="mt-auto flex justify-end">
+							<a
+								href={`/stops/${stop.id}/schedule`}
+								class="inline-block rounded-lg border border-green-500 bg-green-500 px-3 py-1 text-sm font-medium text-white shadow-md transition duration-200 ease-in-out hover:bg-green-600"
+								target="_blank"
+							>
+								View Schedule
```
Comment (aaronbrethorst) [inline src/components/stops/StopPane.svelte:121]: wrap in `t()`

File: src/routes/stops/[stopID]/schedule/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+		} finally {
+			loading = false;
+		}
+	}
+
+	function handleScheduleForStopResponse(scheduleForStop) {
+		schedulesMap.clear();
+		routeReference.clear();
+
+		if (!scheduleForStop.entry.stopRouteSchedules.length) {
```
Comment (aaronbrethorst) [inline src/routes/stops/[stopID]/schedule/+page.svelte:52]: 👍

File: src/routes/stops/[stopID]/schedule/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+	<div class="mx-auto max-w-7xl overflow-y-auto p-5" style="max-height: calc(100vh - 100px);">
+		<StopDetailsHeader {stopName} {stopId} {stopDirection} />
+
+		<div class="flex flex-col gap-6 md:flex-row">
+			<div class="md:w-1/3">
+				<h2 class="mb-4 text-xl font-semibold text-gray-800">Select Date</h2>
+				<div class="rounded-lg border border-gray-300 bg-white p-4 shadow">
+					<Datepicker inline bind:value={selectedDate} />
+					<p class="mt-4 text-sm text-gray-600">
+						Selected date: {selectedDate
```
Comment (aaronbrethorst) [inline src/routes/stops/[stopID]/schedule/+page.svelte:140]: `t()` — here and elsewhere in the file

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
This is looking great! I have noted a couple minor changes to make before merging. Also, now that I see this schedule page with its "Show real time information" button, it feels clear to me that these should be unified in some way. I'll open a new task for that. Bravo!

File: src/components/schedule-for-stop/scheduleAccordionItem.svelte (hunk 2 above)
Comment (Ahmedhossamdev) [reply src/components/schedule-for-stop/scheduleAccordionItem.svelte:1]: Yes

File: src/routes/stops/[stopID]/schedule/+page.svelte (hunk 5 above)
Comment (Ahmedhossamdev) [reply src/routes/stops/[stopID]/schedule/+page.svelte:140]: Yes, if everything works out in the end, I was going to add the localization.

File: src/components/ArrivalDeparture.svelte [hunk 6]
```diff
@@ -38,7 +38,7 @@
 			};
 		} else if (scheduledDiff <= 0) {
 			return {
-				status: 'on time',
+				status: `${$t('status.on_time')}`,
```
Comment (aaronbrethorst) [inline src/components/ArrivalDeparture.svelte:41]: I'll still merge this, but is there any reason this isn't expressed as:

```
status: $t('status.on_time'),
```

isn't the outer template string redundant?

**Review by aaronbrethorst — APPROVED** [review]
👍

File: src/components/ArrivalDeparture.svelte (hunk 6 above)
Comment (Ahmedhossamdev) [reply src/components/ArrivalDeparture.svelte:41]: you're right I will update it in the upcoming PR.

**Pull request merged**